app/addons/databases/stores.js

```diff
@@ -72,13 +72,19 @@ var DatabasesStore = FauxtonAPI.Store.extend({
   },
 
   getDatabaseNames: function () {
-    if (this._backboneCollection) {
-      return _.map(this._backboneCollection.toJSON(), function (item, key) {
-        return item.name;
-      });
-    } else {
+    if (!this._backboneCollection || !this._backboneCollection.length) {
```

Comment:
It would be kind of nice to remove the backbone collection as well and just have a json list of databases. But maybe we do that in the next step.